src/cron-schedule.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * Represents a cron schedule
+ */
+export class CronSchedule {
```

Comment:
Let's rename this to `Cron`. Then we can rename the `custom` function to `schedule`, to have `Cron.schedule()`.

Reply:
Will do.